workflow/usage.rst

```diff
@@ -155,7 +155,19 @@ like this:
     The ``type`` (default value ``single_state``) and ``arguments`` (default value ``marking``)
     attributes of the ``marking_store`` option are optional. If omitted, their default values
     will be used.
+    
+.. note::
+
+    If a ``multiple_state`` is used, the persisted state type should be an ``array``, 
+    using Doctrine, you can use ``json`` or ``json_array``. 
+
+    If a ``single_state`` is used, the persisted state type could be a string, 
+    if you need to store an int (due to DB limitations and/or project constraints),
+    be aware that you need to implement :class:`Symfony\\Component\\Workflow\\MarkingStore\\MarkingStoreInterface`
+    in order to transform the int back to something understandable by Workflow.
 
+    If something else is used, you're in charge of choosing the best storage format.
```

Comment:
Does the previous tip related to MarkingStoreInterface apply in this case too? If yes, we must merge these two phrases.

Reply:
@javiereguiluz yes it's apply to. So it should be merged

Reply:
@Guikingone could you merge theses phrase? Thanks.

After that, IMHO, we will be OK to merge it

Reply:
Improved, to be checked 🙂